.github/workflows/ci.yml

```diff
@@ -182,8 +182,9 @@ jobs:
         run: |
           if [[ -n ${MODULES_ARG} ]]; then
             echo "Running modules: ${MODULES_ARG}"
-            mvn -fae -V -B -s .github/mvn-settings.xml -fae -Dall-modules -pl $MODULES_ARG clean verify -Dnative \
-                        -Dinclude.quarkus-cli-tests -Dts.quarkus.cli.cmd="${PWD}/quarkus-dev-cli"
+            mvn -fae -V -B -s .github/mvn-settings.xml -fae -Dall-modules -Dquarkus.container-image.build=false \
```

Comment:
Why is `-Dquarkus.container-image.build=false` needed to be added to the GH actions here and in the other places?

Reply:
Comment above the `mvn` with some explanation should be present?

Is it because GH Action runner is container and you can't invoke docker build from it?

Reply:
If this is about windows not having containers available, the property should be set in pom.xml of the module, there is already profile for Windows